Sources/StreamChatUI/UIConfig+MessageList.swift

```diff
@@ -12,6 +12,13 @@ public extension _UIConfig {
         public var defaultMessageCell: _СhatMessageCollectionViewCell<ExtraData>.Type =
             _СhatMessageCollectionViewCell<ExtraData>.self
 
+        /// A `UIView` subclass with animated 3 dots for indicating that user is typing.
+        public var typingIndicatorView: _ChatTypingIndicatorView<ExtraData>.Type = _ChatTypingIndicatorView<ExtraData>.self
+
+        /// A `UIView` subclass which serves as container for `typingIndicator` and `UILabel` describing who is currently typing
+        public var typingIndicatorOverlayView: _ChatMessageListIndicatorOverlayView<ExtraData>.Type =
+            _ChatMessageListIndicatorOverlayView<ExtraData>.self
+
```

Comment:
Please check out this document and make sure the changes to UIConfig follow it https://www.notion.so/UI-Config-Rules-Re-Think-5f58357b1f324879927aaeea1aa980a0